### DISCUSSION TITLE
Fragment database used

### DISCUSSION THREAD

**Discussion by DrrDom** [ale94mleon/MolDrug#6] [Ideas]
Hi Alejandro,

I'm a developed of CReM and I noticed that you use a fragment database with suffix sc2 in examples in documentation. I would not recommend to use it, use the one with suffix sa2. SC scores are not very reasonable to estimate synthetic accessibility and those databases were left for historical reasons. Using sa2 database will result in more meaningful molecules. Just a recommendation from my experience.
You may also ask questions by email, if you will have them - pavel_polishchuk@ukr.net.

Pavel.

**Comment by ale94mleon** [comment]
Hi Pavel,
Thank you so much for your comment. Again, congrats for the [CReM](https://github.com/DrrDom/crem) implementation! I just used the smaller data base for the tutorial and the tests in order to do not spend too much time during download. But I will add your comment on the [documentation of moldrug](https://moldrug.readthedocs.io/en/latest/). Thanks again!

**Comment by ale94mleon** [comment]
I already added some comment in the docs. See [here](https://moldrug.readthedocs.io/en/latest/source/note_on_crem.html)
